plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/starlark/references/StarlarkNamedArgumentReference.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.bazel.languages.starlark.references
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import org.jetbrains.bazel.languages.starlark.psi.expressions.arguments.StarlarkNamedArgumentExpression
+import org.jetbrains.bazel.languages.starlark.psi.functions.StarlarkKeywordVariadicParameter
+
+class StarlarkNamedArgumentReference(element: StarlarkNamedArgumentExpression, range: TextRange) :
+  StarlarkArgumentReference(element, range) {
+  override fun resolve(): PsiElement? {
```

Comment:
**suggestion (performance):** Consider optimizing the parameter search in the resolve() method

The current implementation performs two separate find operations on the parameters list. Consider combining these into a single pass for better performance, especially with large parameter lists.